tests/python/multidevice/test_communication.py

```diff
@@ -53,36 +53,26 @@ def test_allreduce(multidevice_test):
     class Model(FusionDefinition):
         def definition(self):
             self.inp = self.define_tensor(
-                (-1, -1, -1), contiguity=True, dtype=DataType.Float
+                (-1, -1), contiguity=True, dtype=DataType.Float
```

Comment:
Any reason behind changing the test case?

Reply:
Because I'm unsure about reorder-sharded-axis

Reply:
Allreduce should be unaffected by that presegmentation pass. The affected collectives would be anything involving a gather or scatter

Reply:
Done!